site-packages/integralstor_utils/system_date_time.py

```diff
@@ -57,11 +57,11 @@ def update_date_time(system_date, system_time, system_timezone):
 def get_current_timezone():
     return_dict = {}
     try:
-        cmd = "timedatectl | grep 'Time' | grep -oE '(\S+/.+$)'"
+        cmd = "timedatectl | grep 'Time' | grep -oE '(:.+$)'"
         system_timezone, err = command.get_command_output(cmd, shell=True)
         if err:
             raise Exception(err)
-        return_dict['system_timezone'] = system_timezone[0]
+        return_dict['system_timezone'] = system_timezone[0][2:]
```

Comment:
@ankit-25 I don't see any difference this commit achieves. Is there any particular reason for the change in grep pattern? The earlier one seems cleaner than this.